hack/run-performance-tests.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Can you reuse this script https://github.com/G-Research/yunikorn-history-server/blob/main/hack/run-tests.sh ?

Reply:
updated